azuki-core/src/main/kotlin/com/anaplan/engineering/azuki/core/runner/ImplementationInstance.kt

```diff
@@ -188,14 +188,15 @@ class JarImplementationInstance<
 
             override fun run() {
                 val implementations = Implementation.locateImplementations<AF, CF, QF, AGF>()
+                    .filter { it.name.split("-").first() == implementationName }
```

Comment:
:pick: if it's not feasible to document in readme can we have a comment here that says if you want to use the same jar for multiple impls, you will need to rename it appropriately